### PR TITLE
API-7419 Add include_customers_without_chats filter to list_customers

### PR DESCRIPTION
**Pull request by janowsiany**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7419-include_customers_without_chats-list_customers)
- [Jira](https://livechatinc.atlassian.net/browse/API-7419)

### CODE REVIEW

**Review by kogierman — APPROVED** [review]

File: content/messaging/agent-chat-api/v3.3/index.mdx [hunk 1]
```diff
@@ -2213,6 +2213,7 @@ All parameters are optional.
 | `filters.created_at.<date_range_filter_type>`**\*\*\*\***                      | `object`  |                           |
 | `filters.agent_last_event_created_at.<date_range_filter_type>` **\*\*\*\***    | `object`  |                           |
 | `filters.customer_last_event_created_at.<date_range_filter_type>` **\*\*\*\*** | `object`  |                           |
+| `filters.include_customers_without_chats`                                      | `boolean` |                           |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.3/index.mdx:2216]:
```suggestion
| `filters.include_customers_without_chats`                                      | `bool` |                           |
```

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/v3.3/index.mdx:2216]: I think we use 'bool'.

File: content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx [hunk 2]
```diff
@@ -2522,6 +2522,7 @@ All parameters are optional.
 | `filters.created_at.<date_range_filter_type>`**\*\*\*\***                      | `object`  |                           |
 | `filters.agent_last_event_created_at.<date_range_filter_type>` **\*\*\*\***    | `object`  |                           |
 | `filters.customer_last_event_created_at.<date_range_filter_type>` **\*\*\*\*** | `object`  |                           |
+| `filters.include_customers_without_chats`                                      | `boolean` |                           |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx:2525]:
```suggestion
| `filters.include_customers_without_chats`                                      | `bool` |                           |
```

**Review by ale-kacperczyk — COMMENTED** [review]
Are we mentioning it in the Changelog?

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**